wordpress-mobile/WordPress-iOS
# Issue/7081 Post Editor: Import media from other applications

This partially addresses [7081](https://github.com/wordpress-mobile/WordPress-iOS/issues/7081).

For iOS11+, this feature adds a new 'Other Apps' option to the Post editor > add media functionality by utilizing `UIDocumentPickerViewController` to enable users to add photos and video from other applications.

![editor_simulator](https://user-images.githubusercontent.com/1816888/31239325-f668aeaa-a9b9-11e7-82f9-d128027ef270.gif)

**To test:**
- Prerequisite: any device must have external media to select from. Ex: iCloud enabled, Google Drive installed.
- Run the app on iOS11+ devices. Edit a post. Tap the add media button and verify 'Other Apps' appears the menu.
- Verify you can add photos and videos from a selected location.
- Run the app on <iOS11 devices. Verify 'Other Apps' does not appear in the menu.

Needs review: @frosty
